specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2685 @@
+      "type": "object",
+      "properties": {
+        "accountName": {
+          "description": "The name of the account that is set as the default.",
+          "type": "string"
+        },
+        "resourceGroupName": {
+          "description": "The resource group name of the account that is set as the default.",
+          "type": "string"
+        },
+        "scope": {
+          "description": "The scope object ID. For example, sub ID or tenant ID.",
+          "type": "string"
+        },
+        "scopeTenantId": {
+          "description": "The scope tenant in which the default account is set.",
+          "type": "string"
+        },
+        "scopeType": {
+          "description": "The scope where the default account is set.",
+          "enum": [
+            "Tenant",
+            "Subscription"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "scopeType",
+            "modelAsString": true
+          }
+        },
+        "subscriptionId": {
+          "description": "The subscription ID of the account that is set as the default.",
+          "type": "string"
+        }
+      }
+    },
+    "BatchFeatureRequest": {
+      "description": "Feature request model",
+      "type": "object",
+      "properties": {
+        "features": {
+          "description": "Set of features",
+          "uniqueItems": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "BatchFeatureStatus": {
+      "description": "List of features with enabled status",
+      "type": "object",
+      "properties": {
+        "features": {
+          "description": "Features with enabled status",
+          "type": "object",
+          "additionalProperties": {
+            "type": "boolean"
+          }
+        }
+      }
+    },
+    "KafkaConfiguration": {
+      "description": "The configuration of the event streaming service resource attached to the Purview account for kafka notifications.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/KafkaConfigurationProperties",
+          "description": "Gets or sets the kafka configuration properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "KafkaConfigurationProperties": {
+      "description": "The kafka configuration properties of the event streaming service attached to the Purview account for kafka notifications.",
+      "type": "object",
+      "properties": {
+        "consumerGroup": {
+          "description": "Consumer group for hook event hub.",
+          "type": "string"
+        },
+        "credentials": {
+          "$ref": "#/definitions/Credentials",
+          "description": "Credentials to access event hub."
+        },
+        "eventHubResourceId": {
```

Comment:
Is it an azure resource ID? If so, please add "format": "arm-id" and "x-ms-arm-id-details".

https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-arm-id-details

Reply:
Yes, it is ARM resource identifier.

Reply:
Added